packages/client-api-schema/src/generated-schema.json

```
@@ -1401,6 +1406,11 @@
       ],
       "type": "object"
     },
+    "Uint256": {
+      "description": "Uint256",
+      "pattern": "^0x([a-fA-F0-9]{64})$",
```

I don't think this needs to be 32 bytes. 
```suggestion
      "pattern": "^0x([a-fA-F0-9]{0,64})$",
```

---

See below comment (this file autogenerated anyway)